scripts/download_data.py

```diff
@@ -21,4 +16,51 @@
 
     gds.get_path("geoda.natregimes")
     gds.get_path("nybb")
-    gds.get_path('geoda health')
+    gds.get_path("geoda health")
+
+
+with suppress(ImportError):
```

Comment:
``` python
hvs.nyc_taxi_remote("download-only")  # return None or path
```

I think we should add something like this in hvsampledata to avoid all of this boilerplate. This will likely also make it so we can move pyarrow back in `pixi.toml`

Reply:
The goal of the script is to pre-download the data, so it doesn't fail in CI tests later on, and to avoid download bars in examples.

The goal of the code was likely to download it, without the latest hvsampledata.